recipes/libpng/all/conanfile.py

```diff
@@ -1,7 +1,11 @@
-from conans import ConanFile, CMake, tools
 import os
+from conans import CMake, tools
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan.tools.files import apply_conandata_patches, get, rmdir, replace_in_file
+from conan.tools.build.cross_building import cross_building
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.51.2"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
1.50.0 is enough here. Version is available since 1.47.0 if IIRC

Reply:
```suggestion
required_conan_version = ">=1.50.2"
```